### ISSUE TITLE
Checkpoint as output

### ISSUE THREAD

**Issue by mah-asd** [ashawkey/torch-ngp#92]
Hi, 
Thank you so much for providing this excellent implementation. 
I have a question about the trained network, how can I save it as checkpoints or other formats?

**Comment by ashawkey** [comment]
@mah-asd Hi, the checkpoints are automatically saved in `<your workspace>/checkpoints/ngp*.pth` for the CMD mode. For the GUI mode, you can click the `save` button to save it to the above location.

**Comment by mah-asd** [comment]
Yes, You're right, thank you for answering. 

Do you think it is possible to use the trained torch-ngp to find the camera poses for images which are not used for training?

**Comment by ashawkey** [comment]
Do you mean the functionality of [iNeRF](https://yenchenlin.me/inerf/)? I think that would require the optimization of the camera pose, but I haven't implemented it now. Of course, You can try to implement it by yourself!

**Comment by mah-asd** [comment]
Yes, thank you for your answer, I'm trying to render a new image based on a trained network and an arbitrary ray. 
here is the code I need to run: 

model = NeRFNetwork(encoding = "hashgrid",bound=opt.bound,cuda_ray = opt.cuda_ray,density_scale = 1,min_near = opt.min_near,density_thresh = opt.density_thresh,bg_radius = opt.bg_radius,)
criterion = torch.nn.MSELoss(reduction='none')
optimizer = lambda model: torch.optim.Adam(model.get_params(opt.lr), betas=(0.9, 0.99), eps=1e-15)
scheduler = lambda optimizer: optim.lr_scheduler.LambdaLR(optimizer, lambda iter: 0.1 ** min(iter / opt.iters, 1))
metrics = [PSNRMeter(),]

trainer = Trainer('ngp', opt, model, device=device, workspace=opt.workspace, optimizer=optimizer, criterion=criterion, 
ema_decay=0.95, fp16=opt.fp16, lr_scheduler=scheduler, scheduler_update_every_step=True, metrics=metrics, 
use_checkpoint = opt.ckpt, eval_interval=50)  

outputs = model.render(rays_o, rays_d, **vars(opt))  # rays_o, rays_d are arbitrary by (B, N, 3) size 

but as an output, I'm getting zero value for output, am I missing a part?
